src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/parameters/Base64EncodedStringParameterValue/Base64EncodedStringParameterValue.jelly

```diff
@@ -27,6 +27,9 @@ THE SOFTWARE.
         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
         xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
         <f:entry title="${it.name}" description="${it.description}">
+            <div name="parameter" description="${it.description}">
```

Comment:
@rinrinne 
> If you are correct, you must show the reason why Rebuild Plugin can deal with NodeParameterValue correctly though NodeLabel Parameter Plugin which provides NodeParameterValue does not have any RebuildParameterProvider extended class.

##### Why it works?
When no provider is available, Rebuild plugin use an existing view provided by Rebuild plugin itself.
Please see code [here](https://github.com/jenkinsci/rebuild-plugin/blob/master/src/main/java/com/sonyericsson/rebuild/RebuildAction.java#L468).

##### Why it is not good anymore?
It is recommended to set values to `RebuildParameterPage` by implementing `RebuildParameterProvider` according to its [documentation](https://github.com/jenkinsci/rebuild-plugin/blob/master/src/main/java/com/sonyericsson/rebuild/RebuildParameterProvider.java#L49).
All page files under resource folder of Rebuild should be internal. Resource folder of Rebuild plugin should not be a "warehouse" for page files from every single plugin that provides support for Rebuild.

##### Why didn't the committer follow recommended instruction?
Please pay attention to the following dates:
* `RebuildParameterProvider` was first introduced by
[this commit](https://github.com/jenkinsci/rebuild-plugin/commit/9da2497ab558cc19dfe63e12b8adbcb89a7c2000) on Oct 6, 2013
* `NodeParameterValue.jelly` was added to Rebuild plugin by [this commit](https://github.com/jenkinsci/rebuild-plugin/commit/32d15659e677f7c22bafd05cee4e2fb2d143d6a4) on Jul 8, 2012

This means when `NodeParameterValue.jelly` was added to Rebuild plugin, there was no extension point provided by Rebuild yet. Adding a jelly file directly to Rebuild was probably  the only way (or hack) to provide Rebuild plugin support at the time.